src/bot.js

```diff
@@ -165,6 +165,46 @@ function addLFG(MESSAGE) {
         });
 }
 
+function endSession(message){
+    var author = message.author,
+        guild_id = message.guild.id,
+        role_id = '',
```

Comment:
You also store the role object itself, in order to delete it. You can obtain the id from that later as well.